### ISSUE TITLE
fastai: wandb not logging fastai validation loss with default args

### ISSUE THREAD

**Issue by eugeneware** [wandb/client#442]
`wandb --version && python --version && uname`

* Weights and Biases version: 0.8.5
* Python version: 3.7.3
* Operating System: Linux
* fast.ai version: 1.0.55

### Description

When you use the `fastai` callback `WandbCallback` with the default arguments you get some errors and the validation loss is not logged.

### What I Did

1. Run the following code in a jupyter notebook
```
import wandb
import fastai
from wandb.fastai import WandbCallback
from fastai.vision import *
from functools import partial
print(f'wandb version: {wandb.__version__}, fastai version: {fastai.__version__}')
wandb.init(project="fastai-test")
path = untar_data(URLs.MNIST_SAMPLE)
data = ImageDataBunch.from_folder(path)
learn = cnn_learner(data, models.resnet18, metrics=accuracy, callback_fns=WandbCallback)
learn.fit_one_cycle(1, 1e-2)
```

2. You'll get the following error:

![image](https://user-images.githubusercontent.com/38154/61618701-b7d63080-acb0-11e9-9eb1-9de4addaa9cd.png)

```
/home/fastai/anaconda3/envs/wandbtest/lib/python3.7/site-packages/fastai/callbacks/tracker.py:50: UserWarning: <class 'wandb.fastai.WandbCallback'> conditioned on metric `val_loss` which is not available. Available metrics are: train_loss, valid_loss, accuracy
  warn(f'{self.__class__} conditioned on metric `{self.monitor}` which is not available. Available metrics are: {", ".join(map(str, self.learn.recorder.names[1:-1]))}')
```

3. To fix it you can tell it to monitor `valid_loss` by changing the callback to be:

``` python
learn = cnn_learner(data, models.resnet18, metrics=accuracy, callback_fns=partial(WandbCallback, monitor='valid_loss'))
```

I'm guessing that the name of the loss changed with fast.ai at some point.

**Comment by vanpelt** [comment]
We'll take a look this week

**Comment by borisdayma** [comment]
This is correct, the default "monitor" value was changed from "val_loss" to "valid_loss".
The callback will be updated to reflect that change.

**Comment by eugeneware** [comment]
Thanks @borisdayma !

**Comment by austinmw** [comment]
`WandbCallback` callback also seems not compatible with `SaveModelCallback` and `learn.load(best_model_name)`. It throws the error:

> Can't pickle local object 'TorchGraph.hook_torch_modules.<locals>.backward_hook'

**Comment by borisdayma** [comment]
I've not seen this before, do you have a way for me to reproduce it?
Actually by default `save_model=True` in the callback, which means that it saves the model and load the best one (in `wandb/RUN_ID` folder) at the end of training.

**Comment by austinmw** [comment]
Ah, so apparently I should have turned off `SaveModelCallback` and allowed it to save/load the best weights on its own. That works, thanks.
